src/ngraph/ngraph_emitter.cc

```diff
@@ -1636,4 +1668,69 @@ void Emitter::CreateLossOps() {
   };
 }
 
+void Emitter::UnsupportedOps() {
+  for (auto kv : ngraph_op_funcs_) {
+    supported_ops[kv.first] = [](const NodePtr& node) { return true; };
+  }
+  supported_ops["BatchNorm"] = [](const NodePtr& node) {
+    bool out = true;
+    auto shape = TShape_to_NShape(node->inputs_[0]->shape_);
+    if (shape[1] % 8 != 0) {
+      // MXNet outperforms nGraph in this case.
+      out = false;
+    }
+    return out;
+  };
+  supported_ops["LeakyReLU"] = [](const NodePtr& node) {
+    bool out = true;
+    // We haven't yet implemented all activation functions for
+    // LeaklyReLU...
+    const std::string act_type =
+        get_default(node, "act_type", std::string("leaky"));
+    if (act_type != "leaky") {
+      out = false;
+    }
+    return out;
+  };
+  supported_ops["Deconvolution"] = [](const NodePtr& node) {
+    bool out = true;
+    auto num_group = get_default(node, "num_group", 1);
+    if (num_group > 1) {
+      if (ngraph_log_verbose_detail()) {
+        std::cout << "NGRAPH_BRIDGE: Deconvolution with num_group > 1 isn't "
+                     "tested in MXNet."
+                  << std::endl;
+        node->printOpDetails(std::cout);
+        std::cout << std::endl;
```

Comment:
FWIW, `printOpDetails(...)` supplies its own, final newline.  So this will add a blank line.

Reply:
I copied it straight out of Compiler::CheckInNgraph(), wanted to keep the style consistent. I think having blank lines makes the dump it easier to read, but it's not a strong preference.